src/modules/fstab/main.py

```diff
@@ -252,8 +254,8 @@ def generate_fstab_line_info(self, partition):
             return dict(device="UUID=" + partition["uuid"],
                         mount_point=mount_point,
                         fs=filesystem,
-                        options=",".join(["subvol={}".format(partition["subvol"]),
-                                          options]),
+                        options=",".join(["subvol=" + partition["subvol"]],
```

Comment:
Are you sure `partition["subvol"]` is a string here? The previous code would convert implicitly.